boards/firechip/base-workloads/ubuntu-base/linux-config

```diff
@@ -0,0 +1,65 @@
+CONFIG_AS_VERSION=23900
+# CONFIG_ATA is not set
+# CONFIG_BACKLIGHT_CLASS_DEVICE is not set
+CONFIG_CMDLINE="console=ttyS0 console=ttySIF0,3686400 earlycon"
+CONFIG_DEBUG_SECTION_MISMATCH=y
+CONFIG_DEFAULT_HOSTNAME="ucb"
+# CONFIG_DRM is not set
+CONFIG_EXT2_FS=y
+# CONFIG_FB is not set
+# CONFIG_HID_GENERIC is not set
+# CONFIG_HWMON is not set
+# CONFIG_I2C_ALGOBIT is not set
+# CONFIG_I2C_COMPAT is not set
+# CONFIG_I2C_HELPER_AUTO is not set
+# CONFIG_INPUT_MOUSE is not set
+# CONFIG_INPUT_MOUSEDEV is not set
+CONFIG_LD_VERSION=23900
+# CONFIG_MMC is not set
+CONFIG_NR_CPUS=32
+# CONFIG_NVMEM_SYSFS is not set
+# CONFIG_PCI is not set
+# CONFIG_POWER_SUPPLY is not set
+# CONFIG_SCSI is not set
+CONFIG_SERIAL_SIFIVE=y
+# CONFIG_SOC_SIFIVE is not set
+# CONFIG_SPI is not set
+# CONFIG_USB_SUPPORT is not set
+# CONFIG_VHOST_MENU is not set
+CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
+CONFIG_ARCH_HAS_DMA_PREP_COHERENT=y
+CONFIG_ARCH_HAS_SETUP_DMA_OPS=y
+CONFIG_ARCH_HAS_SYNC_DMA_FOR_CPU=y
+CONFIG_ARCH_HAS_SYNC_DMA_FOR_DEVICE=y
+CONFIG_AS_HAS_INSN=y
+CONFIG_CC_HAVE_STACKPROTECTOR_TLS=y
+# CONFIG_CMDLINE_EXTEND is not set
+# CONFIG_CMDLINE_FALLBACK is not set
+CONFIG_CMDLINE_FORCE=y
+CONFIG_DMA_COHERENT_POOL=y
+CONFIG_DMA_DIRECT_REMAP=y
+CONFIG_DMA_NONCOHERENT_MMAP=y
+# CONFIG_EXT2_FS_XATTR is not set
+CONFIG_FONT_AUTOSELECT=y
+# CONFIG_I2C_ALGOPCA is not set
+# CONFIG_I2C_ALGOPCF is not set
+# CONFIG_I2C_SMBUS is not set
+# CONFIG_INITRAMFS_COMPRESSION_BZIP2 is not set
+# CONFIG_INITRAMFS_COMPRESSION_GZIP is not set
+# CONFIG_INITRAMFS_COMPRESSION_LZ4 is not set
+# CONFIG_INITRAMFS_COMPRESSION_LZMA is not set
+# CONFIG_INITRAMFS_COMPRESSION_LZO is not set
+CONFIG_INITRAMFS_COMPRESSION_NONE=y
+# CONFIG_INITRAMFS_COMPRESSION_XZ is not set
+# CONFIG_INITRAMFS_COMPRESSION_ZSTD is not set
+# CONFIG_INITRAMFS_FORCE is not set
+CONFIG_INITRAMFS_ROOT_GID=0
+CONFIG_INITRAMFS_ROOT_UID=0
+CONFIG_RISCV_DMA_NONCOHERENT=y
+CONFIG_RISCV_ISA_ZICBOM=0
```

Comment:
Can you also update the Fedora config as well?

Reply:
done